@xen-orchestra/web/src/pages/settings.vue

```diff
@@ -0,0 +1,215 @@
+              </UiLink>
+            </template>
+          </VtsQuickInfoRow>
+          <VtsQuickInfoRow :label="t('community')">
+            <template #value>
+              <UiLink size="medium" href="https://xcp-ng.org/forum/category/12/xen-orchestra">
+                {{ t('community-name', { name: 'Xen Orchestra' }) }}
+              </UiLink>
+            </template>
+          </VtsQuickInfoRow>
+          <VtsQuickInfoRow :label="t('documentation')">
+            <template #value>
+              <UiLink size="medium" href="https://docs.xen-orchestra.com">
+                {{ t('documentation-name', { name: 'Xen Orchestra' }) }}
+              </UiLink>
+            </template>
+          </VtsQuickInfoRow>
+        </VtsQuickInfoColumn>
+        <VtsQuickInfoColumn>
+          <div class="typo-h6">{{ t('xcp-ng') }}</div>
+          <VtsQuickInfoColumn>
+            <VtsQuickInfoRow :label="t('news')">
+              <template #value>
+                <UiLink size="medium" href="https://xcp-ng.org/blog/">
+                  {{ t('news-name', { name: 'XCP-ng' }) }}
+                </UiLink>
+              </template>
+            </VtsQuickInfoRow>
+            <VtsQuickInfoRow :label="t('community')">
+              <template #value>
+                <UiLink size="medium" href="https://xcp-ng.org/forum">
+                  {{ t('community-name', { name: 'XCP-ng' }) }}
+                </UiLink>
+              </template>
+            </VtsQuickInfoRow>
+            <VtsQuickInfoRow :label="t('documentation')">
+              <template #value>
+                <UiLink size="medium" href="https://xcp-ng.org/docs/">
+                  {{ t('documentation-name', { name: 'XCP-ng' }) }}
+                </UiLink>
+              </template>
+            </VtsQuickInfoRow>
+            <VtsQuickInfoRow :label="t('support')">
+              <template #value>
+                <UiLink size="medium" href="https://xcp-ng.com/">
+                  {{ t('pro-support', { name: 'XCP-ng' }) }}
+                </UiLink>
+              </template>
+            </VtsQuickInfoRow>
+          </VtsQuickInfoColumn>
+        </VtsQuickInfoColumn>
+      </VtsColumns>
+    </div>
+    <!-- APPEARANCE -->
+    <div class="container">
+      <UiTitle>{{ t('appearance') }}</UiTitle>
+      <VtsColumns>
+        <div class="typo-h6">{{ t('mode') }}</div>
+        <!-- TODO replace this part when the component is available -->
+        <div class="color-modes">
+          <div
+            v-for="colorModeOption in colorModeOptions"
+            :key="colorModeOption"
+            class="color-mode"
+            :class="{ selected: uiStore.colorMode === colorModeOption }"
+            @click="uiStore.colorMode = colorModeOption"
+          >
+            <img
+              v-if="colorModeOption === 'light'"
+              src="../assets/color-mode-light.svg"
+              :alt="t('dark-mode.disable')"
```

Comment:
It seems strange to see "disable dark mode" but not "enable light mode". What do you think?